<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1292</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblOriginalLeft">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>4</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblOriginalRight">
    <property name="geometry">
     <rect>
      <x>648</x>
      <y>4</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtOutputBox">
    <property name="geometry">
     <rect>
      <x>384</x>
      <y>488</y>
      <width>901</width>
      <height>197</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPauseOrResume">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>145</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowDisparityMap">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>540</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Disparity Map</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowStereoParamSetup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>145</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stereo Param Setup</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShow3DReconstruction">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>580</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show 3D Reconstruction</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowInputImages">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>500</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Left/Right</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1292</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
